components/atom/tooltip/src/styles/index.scss

```diff
@@ -114,6 +112,8 @@ $class-target: '#{$base-class}-target';
   }
 
   &--auto {
+    z-index: $z-atom-tooltip;
```

Comment:
why you have changed this rule inclusion?

Reply:
L21 #{$base-class} { // is for all the elements, which might have this z-index rule and not only the position-auto initially rendered